kedro/ipython/__init__.py

```diff
@@ -281,8 +286,12 @@ def _create_cell_with_text(text: str, is_jupyter: bool = True) -> None:
 
 def _print_cells(cells: list[str]) -> None:
     for cell in cells:
-        Console().print("")
-        Console().print(Syntax(cell, "python", theme="monokai", line_numbers=False))
+        if importlib.util.find_spec("rich") is not None:
```

Comment:
This is a second attempt at importing `rich`, and it's actually going to happen for each cell, which is unnecessarily costly.

I think it's a pretty common pattern to just do this stuff once at the top. See above comment.

Reply:
That's a great point, thanks!